src/lib/styles/modal.less

```diff
@@ -0,0 +1,37 @@
+/* TODO: create separate modal file */
+/* Seems like we maybe could use this modal stuff for projected content in the modal but we're not right now. Should I keep it? */
```

Comment:
the stuff we're using on the Publish modal looks like it is styled with deep selectors